1-lab-basicDispatchConsumeEvent/README.md

```diff
@@ -128,15 +130,15 @@ cdk deploy
 #### Invoking Producer Function
 Now we are going to test the Producer function which will trigger an event to Amazon EventBridge and will be consumed by the Consumer function as well as print the logs into CloudWatch. 
 
-- Create a test event by choosing **Test** in the upper right corner
+- Create a test event by clicking **Test** button in the Lambda code editor menubar 
```

Comment:
Updated layout in Lambda console

Reply:
We need to adjust this particular instruction to adapt to recent UI changes. 
Here's what I have in mind:

- Navigate to the **Test** page by clicking the **Test** tab.
- In the Configure test event page, choose "New Event" and in Event template, leave the default Hello World option.
- Enter an Event name and provide an empty template:

Reply:
updated